physionet-django/console/templates/console/application_list.html

```diff
@@ -0,0 +1,34 @@
+<div id="searchitems">
+  <table class="table table-bordered" id="dataTable" width="100%" cellspacing="0">
+    <thead>
+      <tr>
+        <th>Contact Date</th>
+        <th>Reference Email</th>
+        <th>Reference Name</th>
+        <th>From application</th>
+        <th>Options</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for application in all_known_ref %}
+      <tr>
+        <td>{{ application.reference_contact_datetime | date }}</td>
+        <td>{{ application.reference_email }}</td>
+        <td>{{ application.reference_name }}</td>
+        <td>
+            <a href="{% url 'view_credential_application' application.slug %}" target="_blank">{{ application.id }} -- {{ application.user.get_full_name }}</a>
+        </td>
+        <td>
+            <form action="" method="post">
+            {% csrf_token %}
+              <button  name="remove_known_ref" value="{{ application.id }}" type="submit">
```

Comment:
the style of the button isn't consistent with the button style used elsewhere on the site (e.g. at http://localhost:8000/console/storage-requests/). Is this intentional? I think it would be best to be consistent if possible.

Reply:
It is intensional, since KP prefers styles buttons. (http://localhost:8000/console/complete-credential-applications/)

I can add style if u want tho....

Reply:
Ok, let's leave it like this. I think we do want consistency, but looking around the site I don't think we are particularly consistent anyway, so this can be addressed independently.